plugin/src/main/java/com/fourlastor/pickle/PicklePlugin.kt

```diff
@@ -87,12 +91,12 @@ class PicklePlugin : Plugin<Project> {
             project.plugins.all {
                 when (it) {
                     is LibraryPlugin -> {
-                        project.extensions.findByType(LibraryExtension::class.java).run {
+                        project.extensions.findByType(LibraryExtension::class.java)?.run {
```

Comment:
why the `?` here?

Reply:
I was testing with the newer Android Plugin. findByType is nullable now which makes sense. I decided to keep it.

Reply:
Just removed it. It's not so necessary atm